persistence-zookeeper/src/test/scala/com/mesosphere/usi/storage/zookeeper/ZooKeeperPersistenceStoreTest.scala

```diff
@@ -186,7 +186,7 @@ class ZooKeeperPersistenceStoreTest extends UnitTest with ZookeeperServerTest wi
         store.delete(path).futureValue shouldBe path
       }
 
-      "delete a non-existing node does not lead to an exception" in {
+      "delete a non-existing node does not lead to an exception" ignore {
```

Comment:
So what should it be @zen-dog and @kaiwalyajoshi? Should we fail or not when deleting an unknown record?

Reply:
We should **not**. The end result is the same - node is not there. `delete a non-existing node does not lead to an exception` is correct.